account_move_transactionid_import/parser/transactionid_file_parser.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, profile, ftype='csv', extra_fields=None, header=None,
         super().__init__(
             profile, extra_fields=conversion_dict, ftype=ftype, header=header,
             **kwargs)
+        self.support_multi_moves = True
```

Comment:
should be optional?

Reply:
I don't really understand the message of one commit:
 "parser allows to generate 1 account.move for each line in the imported file"
account move should have at least 2 lines, and second is searched/created on statement reconciliation 
Lint need to be checked

Reply:
> should be optional?

Not when using transaction_id parser due to the change from #292 

> "(multi_move_import) parser allows to generate 1 account.move for each line in the imported file"
account move should have at least 2 lines, and second is searched/created on statement reconciliation

The `multi_move_import` flag on the parser allows to generate 1 account.move per line in the imported statement. However resulting account move (for one line) will always include at least two move lines (cf https://github.com/OCA/account-reconcile/blob/12.0/account_move_base_import/models/account_journal.py#L319-L324) and even 3 if there's a commission amount (cf https://github.com/OCA/account-reconcile/blob/12.0/account_move_base_import/models/account_journal.py#L325-L326).

@Tonow-c2c Could you fix the lint and commit @i-vyshnevska 's suggestion ?